Reverse_Linked_List.cs

```diff
@@ -0,0 +1,27 @@
+/**
+ * https://leetcode.com/problems/reverse-linked-list/
+ * Definition for singly-linked list.
+ * public class ListNode {
+ *     public int val;
+ *     public ListNode next;
+ *     public ListNode(int val=0, ListNode next=null) {
+ *         this.val = val;
+ *         this.next = next;
+ *     }
+ * }
+ */
+public class Solution {
+    public ListNode ReverseList(ListNode head) {
+        ListNode ncab = null;      
```

Comment:
What does `ncab` mean? Nueva cabeza? Super small but it should be `nCab` in that case.